rectangle.js

```diff
@@ -0,0 +1,34 @@
+function area(rectangle) {
+  var width = rectangle.width;
+  var height = rectangle.height;
+
+  return width * height;
+	}
+
+function perimeter(rectangle) {
+  var width = rectangle.width;
+  var height = rectangle.height;
+
+  return (width * 2) + (height * 2);
+}
+
+function diagonal(rectangle) {
+  var width = rectangle.width;
+  var height = rectangle.height;
+
+  return Math.round(Math.sqrt((width * width) + (height * height)));
+}
+
+function isSquare(rectangle) {
+  var width = rectangle.width;
```

Comment:
Nice job! This is a case where I think it would be totally clean and readable to just `return rectangle.width === rectangle.height`